common/djangoapps/edxmako/apps.py

```diff
@@ -0,0 +1,22 @@
+
+from django.apps import AppConfig
+from django.conf import settings
+from . import add_lookup, clear_lookups
+
+
+class EdxMakoConfig(AppConfig):
+    name = 'edxmako'
+    verbose_name = "edX Mako Templating"
+
+    def ready(self):
+        """
+        Setup mako lookup directories.
+
+        IMPORTANT: This method can be called multiple times during application startup. Any changes to this method
```

Comment:
Since this can run multiple times would it be advantageous to add a flag here so the lookups only get added once?

Reply:
Could save us some startup time.

Reply:
It's more likely that this comment is a relic of a different time. In its current location, this code should only be run once AFAIK. I should just remove the comment.

@jmbowman These ready() methods are only called a single time at project startup, correct?

Reply:
@doctoryes Usually, but it's not absolutely guaranteed; see the [note at the end of the `ready()` docs](https://docs.djangoproject.com/en/1.8/ref/applications/#django.apps.AppConfig.ready)

Reply:
OK - since it doesn't appear to happen in regular app startup (the docs indicate only in testing when mucking with `INSTALLED_APPS`), I'm going to leave the comment and forgo adding a flag.